packages/nextra-theme-docs/src/mdx-theme.tsx

```diff
@@ -206,29 +219,123 @@ const Summary = ({
     h5: createHeaderLink('h5', context),
     h6: createHeaderLink('h6', context),
-    a: ({ href = '', ...props }): ReactElement => (
-      <Anchor href={href} newWindow={href.startsWith('https://')} {...props} />
+    ul: ({ children, ...props }: ComponentProps<'ul'>) => (
+      <ul className="ml-6 mt-6 list-disc first:mt-0" {...props}>
+        {children}
+      </ul>
+    ),
+    ol: ({ children, ...props }: ComponentProps<'ol'>) => (
+      <ol className="ml-6 mt-6 list-decimal" {...props}>
+        {children}
+      </ol>
+    ),
+    li: ({ children, ...props }: ComponentProps<'li'>) => (
+      <li className="my-2" {...props}>
+        {children}
+      </li>
+    ),
+    blockquote: ({ children, ...props }: ComponentProps<'blockquote'>) => (
+      <blockquote
+        className="mt-6 first:mt-0 border-l-2 border-gray-300 pl-6 italic text-gray-700 dark:border-gray-700 dark:text-gray-400"
+        {...props}
+      >
+        {children}
+      </blockquote>
+    ),
+    hr: ({ children, ...props }: ComponentProps<'hr'>) => (
+      <hr className="my-8 dark:border-gray-900" {...props}>
+        {children}
+      </hr>
+    ),
+    a: A,
+    table: ({ children, ...props }: ComponentProps<'table'>) => (
+      <table className="mt-6 first:mt-0 p-0" {...props}>
+        {children}
+      </table>
+    ),
+    p: ({ children, ...props }: ComponentProps<'p'>) => (
+      <p className="mt-6 first:mt-0" {...props}>
+        {children}
+      </p>
+    ),
+    tr: ({ children, ...props }: ComponentProps<'tr'>) => (
+      <tr
+        className={cn(
+          'm-0 border-t border-gray-300 p-0 dark:border-gray-600',
+          'even:bg-gray-100 even:dark:bg-gray-600/20'
+        )}
+        {...props}
+      >
+        {children}
+      </tr>
+    ),
+    th: ({ children, align, ...props }: ComponentProps<'th'>) => (
+      <th
+        className={cn(
+          'text-left m-0 border border-gray-300 px-4 py-2 dark:border-gray-600 font-semibold',
+          {
+            center: 'text-center',
+            right: 'text-right'
+          }[align as 'center' | 'right']
+        )}
+        align={align}
+        {...props}
+      >
+        {children}
+      </th>
+    ),
+    td: ({ children, align, ...props }: ComponentProps<'td'>) => (
+      <td
+        className={cn(
+          'text-left m-0 border border-gray-300 px-4 py-2 dark:border-gray-600',
+          {
+            center: 'text-center',
+            right: 'text-right'
+          }[align as 'center' | 'right']
+        )}
+        align={align}
+        {...props}
+      >
+        {children}
+      </td>
     ),
-    table: Table,
     details: Details,
     summary: Summary
   }
 }
 
 export const MDXTheme = ({
-  children
+  children,
+  isRaw
 }: {
   children: ReactNode
+  isRaw?: boolean
 }): ReactElement => {
   return (
-    <MDXProvider components={getComponents() as any}>{children}</MDXProvider>
+    <MDXProvider components={isRaw ? { a: A } : (getComponents() as any)}>
```

Comment:
this solution I described already here https://github.com/shuding/nextra/pull/628#issuecomment-1203231874 and it's works as expected